extra/3-magic-8-ball.js

```diff
@@ -45,8 +45,12 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {
-  //Write your code in here
+
+function shakeBall(herebetterStartingValue) {
+  answerRandom = Math.floor(Math.random() * 16);
+  return answerRandom;
```

Comment:
```suggestion
  answerRandom = allAnswers[Math.floor(Math.random() * allAnswers.length)];
  return answerRandom;
```
Here you get answerRandom from your array

Reply:
use console.log to meet the condition of one of comments `This should log "The ball has shaken!"`